src/main/java/org/apache/commons/lang3/builder/EqualsBuilder.java

```diff
@@ -731,7 +732,8 @@ public EqualsBuilder append(final Object lhs, final Object rhs) {
             // to be inlined
             appendArray(lhs, rhs);
         } else // The simple case, not an array, just test the element
-        if (testRecursive && !ClassUtils.isPrimitiveOrWrapper(lhsClass)) {
+        if (testRecursive && !ClassUtils.isPrimitiveOrWrapper(lhsClass)
```

Comment:
Needs a // comment IMO

Reply:
is now in the JavaDocs of Reflection#isJavaInternalClass(). That should resolve this and all similar situations.